packages/cxl-ui/src/components/cxl-dashboard-team-stats.js

```diff
@@ -16,6 +16,10 @@ export class CxlDashboardTeamStatsElement extends LitElement {
 
   @property({ type: String, attribute: 'manage-roadmaps-url' }) manageRoadmapsURL = '';
 
+  @property({ type: Boolean, attribute: 'show-progress' }) showProgress = false;
+
+  @property({ type: Boolean, attribute: 'show-stats' }) showStats = false;
+
```

Comment:
I suggest to make it `true` by default as it's the most desired situation. Only in some cases we'd like to hide it.

Reply:
I flipped the logic to make it a bit clearer. Instead of show, it's now hide.